src/cluster.c

```diff
@@ -2277,10 +2286,18 @@ void clusterUpdateSlotsConfigWith(clusterNode *sender, uint64_t senderConfigEpoc
                 }
                 clusterDelSlot(j);
                 clusterAddSlot(sender,j);
+                bitmapClearBit(server.cluster->owner_not_owning_slot, j);
                 clusterDoBeforeSleep(CLUSTER_TODO_SAVE_CONFIG|
                                      CLUSTER_TODO_UPDATE_STATE|
                                      CLUSTER_TODO_FSYNC_CONFIG);
             }
+        } else if (server.cluster->slots[j] == sender) {
+            /* The slot is currently bound to the sender but the sender is no longer claiming it.
```

Comment:
super nitpick: the comment lines are a bit long and not of the same length. maybe it is just me though.

Reply:
Addressed this comment. Fixed the comment width to be 80 characters long from the start of comment.

I have fixed the comment width in cluster.h as well.